src/libs/auth.js

```diff
@@ -227,7 +227,8 @@ export const processUser = (user, isSignInEvent) => {
           givenName: profile.givenName,
           familyName: profile.familyName,
           imageUrl: profile.picture,
-          idp: profile.idp
+          idp: profile.idp,
+          allowAppAccess: profile.allowAppAccess
```

Comment:
This name is confusing- why is "appAccess" tied to Azure or the alpha group?

Reply:
And how does this interact with the `isAzureUser` method? Do we still need that method?

Reply:
Naming is hard -- this claim is slightly different than `isAzureUser`. `isAzureUser` is true if a user has logged in via Azure. `allowAppAccess` is true if a user has logged in via Azure _and_ is a member of the Azure preview allow-list. I see this as kind of temporary -- eventually we'll let anyone login via Azure, but for now product/MSFT wants to limit the set of users who are allowed to log in via Azure. And as I realized last year (and am fixing now), we can't use a Sam group for this special case because we want allowed users to be able to log in and register themselves (and accept ToS). 

Given the above, I'll try to use more descriptive names. :) Maybe `isAzureUser()` and `isAzurePreviewUser()` methods in `auth.js` with some comments?

Reply:
Sounds good-- I see no platform connection with "apps" (there are apps in GCP Terra too).

You should also delete `"alphaAzureGroup": "alpha-azure-feature-group"` from the various config files. However, note that the Sam group is still used in back end code to gate access-- you will need to change that.

Reply:
I'm considering whether we want to (a) globally use B2C as the source of truth for the Azure preview allow-list -- or (b) continue to use Sam groups as we do now and treat `AzurePreview.js` as a very special case since it gates user registration.

The benefit of (a) is we maintain the list in 1 place -- but I worry that it will be less flexible/harder to maintain than Sam group(s) for different features. 

Will think a bit more and maybe gather some more feedback..